### ISSUE TITLE
some firacode fonts' render problems

### ISSUE THREAD

**Issue by ahaoboy** [raphamorim/rio#526]
## There is a separator between the last few characters

```bash
echo -e '\\033[38;2;218;98;125m\\033[48;2;218;98;125;30mwin\\033[38;2;218;98;125m\\033[30mC:\wt \\033[48;2;252;161;125;38;2;218;98;125m\\033[48;2;134;187;216;38;2;252;161;125m\\033[48;2;6;150;154;38;2;134;187;216m\\033[48;2;51;101;138;38;2;6;150;154m\\033[0m\\033[K'

```

rio
![image](https://github.com/raphamorim/rio/assets/19884146/9393c4f9-97f9-45e9-92aa-5e12560abe9c)

windows terminal
![image](https://github.com/raphamorim/rio/assets/19884146/3c161b2c-6d49-4e96-9e9b-8fdcf0b4df69)

```toml
[fonts]
family = "FiraCode Nerd Font Mono"
extras = [{ family = "FiraCode Nerd Font Mono" }]
```

## Display problem of two semicircle characters

rio
![image](https://github.com/raphamorim/rio/assets/19884146/a55af9fd-ffac-4b03-b427-2a58cb752d3f)

windows terminal
![image](https://github.com/raphamorim/rio/assets/19884146/017ed924-50cb-4dd6-8076-fec06d5e5edb)

```

echo  ''

```

Remove the font configuration, the separator will disappear, but there will be problems with drawing the semicircle
```toml
# [fonts]
# family = "FiraCode Nerd Font Mono"
# extras = [{ family = "FiraCode Nerd Font Mono" }]


```
![image](https://github.com/raphamorim/rio/assets/19884146/f720c55b-b0e8-4fce-9215-0c19f26d88a3)

**Comment by jinzhongjia** [comment]
"✗"
this also cannot be rendered !

On Rio:
![image](https://github.com/raphamorim/rio/assets/41784264/99f3b562-85db-460d-b913-c1e849debbfc)
On Windows Terminal:
![image](https://github.com/raphamorim/rio/assets/41784264/23847791-bda3-441c-8f40-edc4b81e6a32)

**Comment by jinzhongjia** [comment]
The problem I reported above has been solved in the latest 0.1.0

**Comment by ahaoboy** [comment]
Nice!

![image](https://github.com/raphamorim/rio/assets/19884146/a6c39633-877f-4165-844f-1907d6434bb3)
